tests/e2e/utils/common.go

```diff
@@ -66,6 +65,59 @@ func GenerateDeploymentBody(deploymentName, image string, port, replica int) (er
 	return nil, temp
 }
 
+//fucntion to generate Pod body
```

Comment:
Comment should be started with the function name. In this case, comment should be like

```
GeneratePodBody generate Pod body by Pod Name and image.
```